core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadResult.java

```diff
@@ -24,6 +25,7 @@
  */
 @JacksonXmlRootElement(localName = "CompleteMultipartUploadResult")
 @JsonPropertyOrder({ "Location", "Bucket", "Key", "ETag" })
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
None of the available fields for `CompleteMultipartUpload` seem be fields which are intended to be empty strings, so I'm okay with this.

Reply:
In fact `<Code/>` and `<Message/>` belong to [error responses](https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html#RESTErrorResponses) and not `CompleteMultipartUpload` directly. Again, change should be fine.

Reply:
Thank you @ZhuTopher ! If you saw any abnormal ser/deser results of json or xml, just let me know, thx!